### PR TITLE
Support for read_only_message_tab

### PR DESCRIPTION
**Pull request by abhishek199-dhn**
###  Summary

Adding a support to enable message tab in the app profile.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/{project_slug}/blob/main/.github/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: src/manifest.ts [hunk 1]
```diff
@@ -18,58 +22,77 @@ export class SlackManifest {
...
+          acc[datastore.name] = datastore.export();
+          return acc;
+        },
+        {},
+      );
+    }
+
+    if (def.features?.appHome) {
+      manifest.features.app_home = {
+        messages_tab_enabled: def.features?.appHome.messagesTabEnabled,
```
Comment (selfcontained) [inline src/manifest.ts:92]: Should we be providing a default value here as well, or is there a reason we want to intentionally not default this?

File: src/manifest.ts [hunk 2]
```diff
@@ -18,58 +22,77 @@ export class SlackManifest {
...
+      manifest.datastores = def.datastores?.reduce<ManifestDataStoresSchema>(
+        (acc = {}, datastore) => {
+          acc[datastore.name] = datastore.export();
+          return acc;
+        },
+        {},
+      );
+    }
+
+    if (def.features?.appHome) {
```
Comment (shapirone) [inline src/manifest.ts:90]: This means that `app_home` won't be set unless the developer specifies something at `features.appHome`

**Review by shapirone — CHANGES_REQUESTED** [review]

File: src/manifest.ts (hunk 1 above)
Comment (shapirone) [reply src/manifest.ts:92]: It seems like we need to default this to `true` or else the Messages tab won't appear at all.

**Review by shapirone — APPROVED** [review]
Thanks @abhishek199-dhn!

**Pull request merged**